src/parcel/packager-config.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+module.exports = {
```

Comment:
How do we keep this is sync with the Adobe I/O Runtime image?

Reply:
one idea is to create an action that reports all the packages that are installed on the container and then either add integration test that checks if the definition is up-to-date, or we fetch the definition before deploy time and use it as reference. it can be smart to also check version ranges, so that we still include our version where needed.